sense_index.go

```diff
@@ -34,6 +34,7 @@ type SenseIndexEntry struct {
     SynsetOffset int       // byte offset into <POS>.data file
     SenseNumber int        // sense number within the <POS>.data for the word
     TagCount int           // number of times the word was tagged in semantic concordance texts
+    synsetPtr *Synset      // back ponter to the underlying synset.
```

Comment:
Shouldn't this be public so that callers can access the synset related to this sense. There is no accessor method on `SenseIndexEntry` to get to the synset either